frontend/src/libraries/wamp/types.ts

```diff
@@ -419,11 +286,16 @@ export type TelemetryRequest = {
 export type ProcedureTypes = {
   "account-info": {
     args: [string];
-    result: AccountBasicInfo;
+    result: AccountBasicInfo | null;
+  };
+  // TODO: seems unused on client side, should we remove it?
```

Comment:
It is the method that we introduced for the UI revamp prototype: #784

Reply:
We should move to `preview environments` as soon as possible, it's so clumsy to first deploy backend and then frontend.